IPython/utils/docs.py

```diff
@@ -0,0 +1,7 @@
+# encoding: utf-8
+
+# Copyright (c) IPython Development Team.
+# Distributed under the terms of the Modified BSD License.
```

Comment:
```suggestion
```

When present, those things are just relegate of the Python 2 and pre-github era, there is no need to add them to new files. We just avoid removing to not pollute history.